include/view.h

```diff
@@ -29,6 +29,18 @@ enum ssd_preference {
 	LAB_SSD_PREF_SERVER,
 };
 
+/**
+ * Directions in which a view can be maximized. "None" is used
+ * internally to mean "not maximized" but is not valid in rc.xml.
+ * Therefore when parsing rc.xml, "None" means "Invalid".
+ */
+enum view_axis {
+	VIEW_AXIS_NONE = 0,
+	VIEW_AXIS_HORIZONTAL = (1 << 0),
+	VIEW_AXIS_VERTICAL = (1 << 1),
+	VIEW_AXIS_BOTH = (VIEW_AXIS_HORIZONTAL | VIEW_AXIS_VERTICAL),
+};
```

Comment:
I like this `VIEW_AXIS_BOTH` variant. This fixes the odd "enum consists of bits and now we have some kind of var which contains a combination of the bits, what type should that var be" issue. Also I didn't know that this actually works (e.g. using enum entries from the same enum). Nice solution.